packages/browser-wallet/src/popup/popupX/pages/Onboarding/IdSubmitted.tsx

```diff
@@ -20,7 +21,7 @@ export default function IdSubmitted() {
             <Page.Top heading={t('yourId')} />
             <Page.Main>
                 <Text.Capture>{t('idSubmitInfo')}</Text.Capture>
-                <IdCard rowsIdInfo={rowsIdInfo} />
+                <IdCard rowsIdInfo={rowsIdInfo} idProviderName="TODO" identityName="TODO" />
```

Comment:
And here

Reply:
We need to hook up the identity issuance first